### DISCUSSION TITLE
How to work with Pimcore?

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#10225] [General]
After the fifth or sixth project on Pimсore, I had a strategic question:
How to work with Pimcore?
a) Take a skeleton and add everything you need with your hands
b) Take ready-made extensions
c) Write your extensions

What advice for a beginner?

**Comment by fashxp** [comment]
It depends on the use cases and requirements and depending on them you will work with all three approaches.
- take a skeleton to have a base application and add custom logic there
- use ready-made extensions where available
- create your own extensions for logic that should be reusable
